.circleci/config.yml

```diff
@@ -27,6 +27,14 @@ executors:
       CACHE_VERSION: v1
     working_directory: ~/react-spectrum
 
+  rsp-xlarge-nodeupdate:
+    docker:
+      - image: cimg/node:16.18.0
```

Comment:
16.18 needed to test modules imports with a custom CSS loader
but I didn't want to introduce that much of a change across the board